shownotes/S01E14-亮小猪：从开发到管理到开发者关系，他都经历了什么.md

```diff
@@ -0,0 +1,47 @@
+---
+num: 'S01E14'
+title: '[亮小猪：从开发到管理到开发者关系，他都经历了什么](https://www.xiaoyuzhoufm.com/episode/xxx)'
+guests: '[亮小猪](https://github.com/shinyzhu)'
+hosts: '[Rick](https://github.com/linuxsuren)、[Wey Gu](https://github.com/wey-gu)、[MINGAI](https://twitter.com/ZMA222222)'
```

Comment:
We only need the GitHub ID. See also https://github.com/opensource-f2f/episode/blob/e3186fec2b06420880c50522e6e2ea9b1ea6bd93/README.tpl#L13

Reply:
https://github.com/PROPELLER-ZMA